_posts/en/newsletters/2022-04-27-newsletter.md

```diff
@@ -0,0 +1,240 @@
+    the consensus rules to add a new feature creates risks for all
+    Bitcoin users---even for people who don't plan to use the new
+    feature---so it's important to provide the non-adopters with public
+    evidence that the feature will be "valuable enough to others [to
+    justify] the risk".
+
+  - Matt Corallo also [argued][corallo ctv cost] that changing consensus
+    creates significant costs for everyone so we should only attempt
+    soft forks when we're sure a proposal provides the maximum value
+    available from a change.  In the case of covenants, Corallo wants to
+    see "the most flexible and useful and hopefully private" design.  He
+    [later said][corallo not clear], "from what I have seen it's not
+    been clear that CTV is really a good option".
+
+  - Russell O'Connor [noted][oconnor wizards] on #bitcoin-wizards IRC
+    that one of the proposed ways of using CTV could not be communicated
+    in an existing Bitcoin address format such as base58check,
+    [bech32][topic bech32], or bech32m.  That method of using CTV via
+    *bare script* (a script that appears directly in a scriptPubKey)
+    would require wallet developers only use bare CTV with their own
+    internal transactions or write special tools to communicate the
+    information usually contained in an address.  Alternatively, wallets
+    wanting to use CTV for some applications (such as [vaults][topic
+    vaults]) could receive payments to a P2TR address which committed to
+    later using bare CTV.
+
+    O'Connor's discussion about the address limitation was
+    [mentioned][towns bare] on list by Towns.  O'Connor
+    [replied][oconnor bare] with additional details and also noted that,
+    if support for bare CTV wasn't part of the BIP119 specification of
+    CTV, he'd advocate for a different design that would be simpler and
+    more composable (easier to combine with other opcodes to produce
+    useful scripts)---although, beyond that, he'd prefer the more
+    general `OP_TXHASH` design (see [Newsletter #185][news185 txhash]).
+    Rubin [replied][rubin bare] with several counterpoints.
+
+  - David Harding [relayed][harding transitory] a concern that CTV might
+    not receive much long-term use, either because uses for it wouldn't
+    manifest or because other covenant constructions would better serve
+    popular uses.  This would leave future consensus code developers
+    with the perpetual burden of maintaining CTV code and analyzing its
+    potential interactions with later proposed consensus changes.
+    Harding suggested temporarily adding CTV to Bitcoin for five years,
+    gathering data during that time about how people used it, and then
+    automatically disabling it unless Bitcoin users five years from now
+    decided it was worth keeping.  No respondents were in favor of this
+    approach, with most of them arguing either that the costs of this
+    approach were too high or the benefits too low.
```

Comment:
I thought an interesting point in the responses here was that it cannot be fully removed because nodes will need to be able to validate the transactions in that period of the blockchain. Perhaps that is covered in "costs too high" already, but if we were to elaborate, that would be the point I'd include.